scopes/pkg/pkg/pkg.docs.md

```diff
@@ -1,35 +1,38 @@
 ---
-description: Bit Component package automation, packing and publishing.
+description: Generates, packs and publishes component packages
 labels: ['packages', 'aspect', 'pkg']
 ---
 
-Pkg aspect automates the creation, packing and publishing of packages of Bit components.
+Bit components can be thought of as a super-set of standard packaged node modules. Each component contains a consumable package in addition to its documentation, history and other information that enables it to be maintained independently.
 
-1. allows users to change (add) properties in the component's package.json
-2. allows other extensions to add new properties to the component's package.json
-3. allows an env to add new properxties to the component's package.json
-4. exposes a pack command to pack a component into a tar suitable for an npm registry
-5. exposes a publish command to publish components to a private registry
-6. utilizes PostTag hook to auto publish components after tag
+Components are not only consumed as standard packages but can also be [published to NPM](/docs/packages/publish-to-npm) or any other package registry (in addition to being 'exported' to a remote scope on a bit server).
 
-## Usage
+The 'pkg' aspect generates, packs and publishes component packages. That includes:
 
-### Configuration
+1. Allowing users to modify properties in the component's `package.json`
+2. Allowing other Bit extensions to add new properties to the component's `package.json`.
+3. Allowing a Bit environment to add new properties to the component's `package.json`.
+4. Exposing a `pack` command to pack a component into a tar suitable for an npm registry
+5. Exposing a `publish` command to publish components to a private registry
+6. Utilizing the `PostTag` hook to auto publish components after tag
 
-#### Workspace configuration
+### Quickstart & configuration
 
-This extension doesn't get any configuration in the workspace level.
+> This aspect is only configurable using the 'variants' workspace API.
 
-#### Variants configuration
+#### Package properties
 
-This extensions gets properties to add to the package.json in the variants config in this format:
+Use the `packageJson` property to add or override the default `package.json` for your component packages.
 
 ```js
 {
   "ui/*": {
     "teambit.pkg/pkg": {
       "packageJson": {
-        "myPropToAdd": "propValue"
+          "name": "@{scope}/{name}",
```

Comment:
keep in mind that once you set the name it won't publish it to bit's registry 
see this code in `publisher.ts`
```js
public shouldPublish(extensions: ExtensionDataList): boolean {
    const pkgExt = extensions.findExtension(PkgAspect.id);
    if (!pkgExt) return false;
    return pkgExt.config?.packageJson?.name || pkgExt.config?.packageJson?.publishConfig;
  }
  ```